packages/installer/cos.sh

```diff
@@ -1403,6 +1406,8 @@ install() {
             --help)
                 usage
                 ;;
+            --silence-hooks)
```

Comment:
I'm totally ok with it, but from a naive perspective my first thought was to attach it to a general `--debug` flag instead of having its own option, what do you think?

Reply:
like if debug is enabled, then show the normal logging from yip, or even set yip to debug, otherwise set it on error by default?

Reply:
> like if debug is enabled, then show the normal logging from yip, or even set yip to debug, otherwise set it on error by default?

yup exactly, so in case if something goes off, someone can just add `--debug`, and we have more detailed logs to debug as well... but that's a thought! if you think it's better of keep them separated I'm fine with. Currently we don't use debug extensively here, but maybe it's something to keep in mind for elemental-cli